src/Stepper/README.md

```diff
@@ -138,7 +138,7 @@ A composition of a stepper with a `FullscreenModal`.
               <HipsterIpsum numParagraphs={2} />
             </Stepper.Step>
 
-            <Stepper.Step eventKey="choose-cats" title="Chooseth a cat">
+            <Stepper.Step eventKey="choose-cats" title="Chooseth a cat" hasError="true" description="This is error text">
```

Comment:
Instead of changing this existing one maybe add a new one for the error state?

Reply:
I have reverted this change because it was out of the scope of this Fix

Reply:
@AhtishamShahid is there any existing item in the readme that illustrates the validation error? If not, it's ok to update the docs I guess.